deepchecks/checks/distribution/__init__.py

```diff
@@ -11,3 +11,4 @@
 """Module containing all data distribution checks."""
 from .trust_score_comparison import *
 from .train_test_drift import *
+from .whole_dataset_drift import *
```

Comment:
@shir22 WDYT?

Reply:
While I agree that "whole" isn't scientific, I think that multivariate drift isn't a good match to this scenario.
Since here we're explicitly talking about Dataset 1 vs. Dataset 2 -> are they drifted one compared to the other, while not necessarily the same dataset, and necessarily all of the features. I think multivariate drift doesn't encapsulate that meaning and may confuse with other options.

Completely open to alternative suggestions, but between the two offered here, whole_dataset_drift is clearer and less confusing IMO.

Reply:
drift_between_datasets is not in our format, and between_datasets_drift is not 100% clear, but it may be better then whole_dataset_drift so I'm willing to roll

Reply:
I don't like either but I have no idea at the moment.

Reply:
I'm leaving it be for the duration of this PR